.kitchen.ci.yml

```diff
@@ -0,0 +1,48 @@
+---
+driver:
+  name: rackspace
+  rackspace_username: <%= ENV['RACKSPACE_USERNAME'] %>
+  rackspace_api_key: <%= ENV['RACKSPACE_API_KEY'] %>
+  rackspace_region: 'lon'
+  flavor_id: 'performance1-1'
+  privileged: true
+  require_chef_omnibus: 12.7.2
+
+provisioner:
+  name: chef_zero
+  data_path: test/shared
+
+platforms:
+  - name: ubuntu-14.04
+    run_list:
+      - recipe[apt]
+
+suites:
+  - name: postgresql
+    data_bags_path: "test/integration/mysql/data_bags"
```

Comment:
Well spotted. So, in the upstream cookbook, the `mysql` suite had MySQL-related data bags, and the `postgres` suite was broken because it had no data bags, so I re-used the supplied MySQL ones. They're actually generic data so should be in a generic place. I'll do that now...